utils/crawlGameSchedule.js

```diff
@@ -0,0 +1,54 @@
+const puppeteer = require("puppeteer");
+
+const crawlGameSchedule = async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  await page.goto("https://www.koreabaseball.com/Schedule/GameCenter/Main.aspx#none;");
+
+  const nextGameDate = await page.evaluate(() => (
+    document.querySelector(".date-txt").textContent
```

Comment:
```suggestion
    document.querySelector(".date-txt").textContent;
```
세미콜론 빠졌는데 왜 ESLint가 알려주지 않은 것인가 😣